src/components/Product/ErrorTracking/index.tsx

```diff
@@ -281,11 +281,19 @@ const comparison = [
     {
         feature: 'Integration with session replays',
         companies: {
-            Sentry: false,
+            Sentry: true,
             LogRocket: true,
             PostHog: true,
         },
     },
+    {
+        feature: 'Integration with feature flags',
+        companies: {
+            Sentry: false,
```

Comment:
they've got flags in open beta iirc

Reply:
<img width="510" alt="Screenshot 2025-04-08 at 17 40 46" src="https://github.com/user-attachments/assets/8a5d4a88-67c4-4a5b-893f-9b09e4ba7dc0" />

Reply:
I'd leave it as flags tbh (all starts to depend on what an integration is... would you count a framework specific SDK)

can just add a note that they have flag integration in open beta and that it relies on external flag provider (which i think is right)

Reply:
we've changed to experiments for now just to avoid further consternation or vagueness -- happy for David and Team to make another change here if needed